src/extract-args-types/extract-arg-types.ts

```diff
@@ -23,27 +23,26 @@ const mapPropToArgsTableProp = (
     prop: Property,
     category: string
 ): ExtendedArgType => ({
-    name: prop.name,
+    name: prop.alias || prop.name,
     description: prop.description,
     defaultValue: prop.defaultValue,
     table: {
         defaultValue: {
-            summary: prop.defaultValue,
-            detail: undefined, // 'defailtValueDetail', todo show details for interfaces and types
+            summary: prop.defaultValue || "---",
```

Comment:
Do we need to render `---`? Probably a thing that `ArgsTable` needs to decide (rendering empty values)

Reply:
Jup, i just wanted to enter something here that clearly indicates to me that there is no defaultValue set. We can replace this as desired.